### PR TITLE
CFINSPEC-5 Added more detailed description about waivers expiration date.

### LINKED ISSUE #5794: Docs: Waiver expiration not explained
# Describe the problem

[Waiver documentation](https://docs.chef.io/inspec/waivers/) doesn't explain when the waiver will expire.

It's unclear if a waiver with an expiry date of `2022-01-11` will expire at `00:00` on the 11th or `23:59` on the 10th.
It also doesn't state if the InSpec respects timezones or calculates everything based on UTC

## Possible Solution

Documentation updated stating the two above problems.

### PR DESCRIPTION
**Pull request by Vasu1105**
Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

## Related Issue

Fixes #5794
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs-chef-io/content/inspec/waivers.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ control_id:
   justification: "reason for waiving this control"
 ```
 
-- `expiration_date` is optional. Absence means the waiver is permanent.
+- `expiration_date` is optional. Absence means the waiver is permanent. If the expiration date is `2019-10-15` then
+  it expires at `00:00` on the 15th of October. This considers the system date and time on which you are executing the inspec command to verify the expiration date.
```
Comment (damacus) [inline docs-chef-io/content/inspec/waivers.md]: Is this 15th of October UTC or an offset based on the computer it's running on?

**Review by damacus — CHANGES_REQUESTED** [review]

File: docs-chef-io/content/inspec/waivers.md (hunk 1 above)
Comment (Vasu1105) [reply docs-chef-io/content/inspec/waivers.md]: The offset is based on the computer it's running on.

Comment (damacus) [reply docs-chef-io/content/inspec/waivers.md]: Cool, can we be explicit about that please?

It's a tad confusing :)

Comment (Vasu1105) [reply docs-chef-io/content/inspec/waivers.md]: I think I covered that by mentioning that it considers the system date and time on which you are executing the inspec command. Could you please suggest what I should change in that?

**Review by damacus — APPROVED** [review]

File: docs-chef-io/content/inspec/waivers.md (hunk 1 above)
Comment (IanMadd) [inline docs-chef-io/content/inspec/waivers.md]: How about this:
```suggestion
- `expiration_date` sets the day that the waiver file will expire in YYYY-MM-DD format. Waiver files expire at 00:00 at the local time of the system on the specified date. Waiver files without an expiration date are permanent. `expiration_date` is optional.
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**